src/components/todo-list.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <ul>
+    <TodoItem 
```

Comment:
You have props in `TodoItem` :  `todo`, `index` but you do not set index prop to the component